test/JDBC/expected/13_5__preparation__sys-assembly_types-vu-prepare.out

```diff
@@ -3,7 +3,7 @@ SELECT * FROM sys.assembly_types
 GO
```

Comment:
We not only need order by clause for this test but for TestSpatial test as well. I could still see a lot of select happening in that test without any ORDER BY clause which is not failing today but might fail in future.

Reply:
Done